backend/promptfoo/create_esg_report_config.yaml

```diff
@@ -0,0 +1,110 @@
+description: "Test ESG Report Prompt"
+
+providers:
+  - id: mistral:mistral-large-latest
+    config:
+      temperature: 0
+
+prompts: file://promptfoo_test_runner.py:create_prompt
+
+tests:
```

Comment:
could we add a few basic tests
for example:
- has the 3 headings "Environmental", "Social" and "Governance"
- for the AWS report the percentages should be easy enough to test since it's 100% environmental
- another test for another report eg kingfisher
- any content that might be wanting to be tested?

Reply:
I've added tests for the first bullet point.

For testing the specific content we expect to see in the report, I would lean towards not locking ourselves into expecting anything particular at the moment. Once this is in we need to take stock of where we are, what we are seeing and present it to Arbdn to get feedback on the direction. Trying to capture specifics in these reports via tests is hard because this could change as we learn more from Arbdn